CRM/MembershipExtras/Job/OfflineAutoRenewal/MultipleInstallmentPlan.php

```diff
@@ -247,7 +268,7 @@ private function copyRecurringLineItems($currentContribution, $nextContribution)
         CRM_MembershipExtras_BAO_ContributionRecurLineItem::create([
           'contribution_recur_id' => $nextContribution['id'],
           'line_item_id' => $newLineItem['id'],
-          'start_date' => $nextContribution['start_date'],
+          'start_date' => $this->paymentPlanStartDate,
```

Comment:
my memory is a bit vague about this, but should the recur line item start date contain the related membership start date , or should  it really be the recur contribution start date as u doing here?

Reply:
It should be the date the recurring contribution starts, which has to do with the cycle day. 

On manage installments screen, the period start date is the earliest membership start date. On that same screen, each line shows the membersip start date, if it's a membership, recurring line start_date if not (eg. donations, events, etc).

On contributions tab, the recurring contribution start date is the date of the first installment (as affected by cycle day). On memberships tab, start date is the membership start date.